src/libraries/sendtohelixhelp.proj

```diff
@@ -34,7 +34,7 @@
         '$(Scenario)' == 'gcstress0xc_zapdisable_heapverify1' or
         '$(Scenario)' == 'gcstress0xc_jitstress1' or
         '$(Scenario)' == 'gcstress0xc_jitstress2' or
-        '$(Scenario)' == 'gcstress0xc_jitminopts_heapverify1'">01:30:00</_workItemTimeout>
+        '$(Scenario)' == 'gcstress0xc_jitminopts_heapverify1'">06:00:00</_workItemTimeout>
```

Comment:
Is this enough for the tests to finish? Do you plan to enable the scheduled runs for the pipelines?

Reply:
It's not, but I'm not sure if that's because of a general incompatibility of GCStress and some tests, or if some are just really, really slow.

I'm not planning to enable scheduled runs unless I can disable enough tests to make it run consistently cleanly.

Reply:
Do I understand correctly that we won't run this scenario as part of default configurations (CI, PR Validation).  If so, then this is OK.

Reply:
Yes, that's correct. These gcstress scenarios are only run in the gcstress pipelines, which aren't run regularly (or at all) currently. Hopefully we can fix the inherent issues with running them and then schedule a weekly run, but that's going to take some work.